packages/zosfiles/src/constants/ZosFiles.messages.ts

```diff
@@ -198,6 +197,13 @@ export const ZosFilesMessages: { [key: string]: IMessageDefinition } = {
         message: "Member(s) downloaded successfully.\nDestination: %s"
     },
 
+    /**
+     * Message indicating that the members contents were truncated due to lrecl
+     * @type {IMessageDefinition}
+     */
+    membersContentTruncated: {
+        message: "Member(s)' contents were truncated due to insufficient record lines. You can view the list of members here: %s"
```

Comment:
Since `lrecl` is Logical Record Length, should this say "insufficient record length"? I believe that would be clearer than "lines" since the issue is the max length of lines not the total number of lines?